plugin.xml

```diff
@@ -75,59 +75,31 @@
       <string>production</string>
     </config-file>
 
-    <!-- Header files -->
-    <header-file src="src/ios/headers/SFAuthenticationManager.h" target-dir="salesforce"/>
```

Comment:
We don't have to explicitly declare headers in our plugin anymore since they're packaged in the framework.